theano/tensor/nnet/ctc.py

```diff
@@ -118,6 +120,7 @@ def __init__(self, compute_grad=True, openmp=None):
         gof.OpenMPOp.__init__(self, openmp=openmp)
 
         self.compute_grad = compute_grad
+        self.openmp = gof.OpenMPOp(self).openmp
```

Comment:
Is this line still useful ? As op inherits from `OpenMPOp`, I thought it should already have an attribute nammed `openmp`. Could you check if everything works when you remove that line ?